container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -2154,45 +2171,62 @@ instance_groups:
     release: app-autoscaler
     properties:
       bosh_containerization:
+        colocated_containers:
+        - loggregator-agent
         run:
           scaling:
             min: 1
             max: 3
             ha: 2
-          memory: 128
+          memory: 1024
```

Comment:
What is the reason behind such a significant change in memory requirements?

Reply:
In the recent app-autoscaler releases,  we enabled local cache to reduce the frequency of database access for metric data, so more memory is required here.